src/web/api/README.md

```diff
@@ -1,22 +1,80 @@
-# Netdata API's
+# Netdata APIs
+
+Netdata provides two APIs for programmatic access to your monitoring infrastructure.
 
 ## Agent API
 
-The complete documentation of the Netdata Agent's REST API is documented in the OpenAPI format [in our GitHub repository](https://raw.githubusercontent.com/netdata/netdata/master/src/web/api/netdata-swagger.yaml). 
+The Netdata Agent REST API provides access to metrics, alerts, and configuration on individual nodes. The complete documentation is available in OpenAPI format.
+
+Explore the Agent API using:
 
-You can explore it using the **[Swagger UI](https://learn.netdata.cloud/api)**, or the **[Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/netdata/netdata/master/src/web/api/netdata-swagger.yaml)**.
+- **[Swagger UI](https://learn.netdata.cloud/api)** - Interactive API explorer
+- **[Swagger Editor](https://editor.swagger.io/?url=https://raw.githubusercontent.com/netdata/netdata/master/src/web/api/netdata-swagger.yaml)** - Edit and test API calls
+- **[OpenAPI Specification](https://raw.githubusercontent.com/netdata/netdata/master/src/web/api/netdata-swagger.yaml)** - Raw OpenAPI YAML
 
 ## Cloud API
 
-Netdata Cloud offers a public REST API for programmatic access to Cloud resources, spaces, war rooms, and nodes.
+The Netdata Cloud REST API provides programmatic access to Cloud resources, spaces, war rooms, and nodes across your infrastructure.
```

Comment:
I _think_ that the _**war** rooms_ lingo is deprecated, perhaps we can drop the war word